test/browser/features/steps/browser_steps.rb

```diff
@@ -119,3 +123,12 @@
     step('the event "device.id" is null')
   end
 end
+
+Then("the Bugsnag-Integrity header is valid") do
+  raw_request = Server.current_request[:request]
+
+  type, value = raw_request['Bugsnag-Integrity'].split(' ')
+
+  assert_equal('simple', type)
+  assert_equal(raw_request.body.bytesize, value.to_i)
```

Comment:
Since this uses the ruby bytesize method we could leverage it further to test that the `@bugsnag/bytesize` module you behaves in the same way for some of the complicated cases.

It would be good to add a feature file and some fixtures that send some combinations of multibyte characters, rather like you have in the unit tests for the module itself.

Reply:
I did verify the original implementation with Ruby's bytesize method, but that is a good idea to make sure it stays equivalent

Should I make a new ticket for that though, as this integration branch won't be merged that soon?

Reply:
Added some MR tests with the examples in 90595bd28d540524a889e0d4678f1402bde7eea3